src/services/ApiService.ts

```diff
@@ -0,0 +1,24 @@
+import { create } from 'apisauce';
+import { SnakecaseSerializer } from 'cerealizr';
+
+import { IFormValues } from '../interfaces/form';
+
+const serializer = new SnakecaseSerializer();
+
+const api = create({
```

Comment:
Create a config folder and put all api init logic inside, to keep only service calls here.